source/common/router/config_impl.h

```diff
@@ -362,8 +362,10 @@ class RouteEntryImplBase : public RouteEntry,
 
   void finalizePathHeader(Http::HeaderMap& headers, const std::string& matched_path,
                           bool insert_envoy_original_path) const;
-  const HeaderParser& requestHeaderParser() const { return *request_headers_parser_; };
-  const HeaderParser& responseHeaderParser() const { return *response_headers_parser_; };
+  const HeaderParser& requestHeaderParser() const { return *route_action_request_headers_parser_; };
+  const HeaderParser& responseHeaderParser() const {
+    return *route_action_response_headers_parser_;
+  };
```

Comment:
Sorry missed this in the previous pass, if we remove `response_headers_to_*` and `request_headers_to_add` at `RouteAction` level (since it is deprecated in this PR) what will be returned in these two getters above? Do you know which part of Envoy makes use of these two getters?

Reply:
Should we just get rid of them?

Reply:
👍